olive/passes/onnx/quantization.py

```diff
@@ -611,23 +618,65 @@ def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassCon
             "algorithm": PassConfigParam(
                 type_=str,
                 default_value=None,
-                description="Available from onnxruntime>=1.17.0 "
-                "Algorithm config for quantization, can be 'DEFAULT', 'HQQ' or 'RTN', 'GPTQ' "
-                "(default DEFAULT when not set). For 4b quantize a model with RTN or GPTQ algorithm. "
-                "Please refer to https://github.com/intel/neural-compressor/blob/master/docs/source/quantization_weight_only.md "  # noqa: E501
+                description="'RTN' and 'GPTQ' are available from onnxruntime>=1.17.0 "
+                "'DEFAULT', 'HQQ' are available from onnxruntime>=1.18.0 (default DEFAULT when not set). "
+                "For 4b quantize a model with RTN or GPTQ algorithm. Please refer to "
+                "https://github.com/intel/neural-compressor/blob/master/docs/source/quantization_weight_only.md "
                 "for more details on weight only quantization using Intel® Neural Compressor. "
                 "For HQQ, please refer to onnxruntime for more details: "
                 "https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/quantization/matmul_4bits_quantizer.py#L102C1-L126C25",
             ),
+            "weight_only_quant_configs": PassConfigParam(
+                type_=dict,
+                default_value=None,
+                description="""Available from onnxruntime>=1.17.0
+                The config is binding to the algorithm with following map:
+                1. "algorithm" is "DEFAULT", by default, the weight_only_quant_configs is:
```

Comment:
I'm kinda confused what is the default value for `algorithm`

Reply:
Default value is `None` and the behavior is same with option `DEFAULT`.
Updated the docs. Could you help take a look? :)

Reply:
For `weight_only_quant_configs`, if it is None, the **default config** of given algorithm will be used.
![image](https://github.com/microsoft/Olive/assets/13343117/fcc93ad2-5a98-45d2-8e15-25e3731c107b)

Reply:
If `weight_only_quant_configs` and `algorithm` are both given, will the Olive ignore `algorithm` but use `weight_only_quant_configs`?

Reply:
Nope. `algorithm` is used to tell which kind of `weight_only_quant_configs` will be used. Otherwise, the quantization would fail.
If both given, and they are matched, it works. If both given, but they are not matched, would fail.